### DISCUSSION TITLE
Support for public cloud

### DISCUSSION THREAD

**Discussion by fexga** [sustainable-computing-io/kepler#873] [General]
Hi,

is it possible to utilize Kepler in the public cloud?

Unfortunately, the Kepler Grafana dashboard is showing "No data" for my deployment in Azure.

Thank you!

**Comment by rootfs** [comment]
yes, we have deployed kepler on public clouds. The reasons of "no data" could be certain configuration issues. please search the previous issues and see if there are similar symptoms and solutions.
